core/src/main/java/io/paradoxical/cassieq/admin/resources/api/v1/AccountResource.java

```diff
@@ -226,14 +206,12 @@ public Response addNewKey(
     @ApiResponses(value = { @ApiResponse(code = 204, message = "Ok"),
                             @ApiResponse(code = 500, message = "Server Error") })
     public Response deleteAccount(@StringTypeValid @PathParam("accountName") final AccountName accountName) {
+
+        lookupAccount(accountName);
```

Comment:
thats valid. lets file an issue to move to a CheckedException system. at least it will be easy to move to checked exceptions having these inplace